metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/ReferenceSequence.java

```diff
@@ -87,11 +91,28 @@
     @ManyToOne(optional = false)
     private Taxonomy taxonomy;
```

Comment:
As discussed with @selvaebi, taxonomy needs to be retrieved inside ReferenceSequence retrieval as well.

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/50/commits/767b4d1a20eda626c547e83b812b3ea353078d00